web/crypto_utils.py

```
@@ -0,0 +1,149 @@
+    def from_db_value(self, value, expression, connection):
+        if value is not None:
+            value = decrypt_value(value)
+        return value
+
+    def to_python(self, value):
+        if isinstance(value, str) and value.startswith("gAAAAA"):  # Likely encrypted
+            value = decrypt_value(value)
+        return super().to_python(value)
+
+
+def anonymize_email(email):
```

Ruff (0.8.2)

Missing return type annotation for public function `anonymize_email`

(ANN201)